ShopifySharp/Entities/OutgoingRequestOptions.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    public class OutgoingRequestOptions
```

Comment:
Would it make sense to rename this class and file to `OutgoingFulfillmentRequestOptions`?